### DISCUSSION TITLE
Is it normal that saying "hello" isn't detected well?

### DISCUSSION THREAD

**Discussion by qwbarch** [snakers4/silero-vad#585] [Q&A] [answered]
I noticed that when I say "hello" the probability is always very low. If I say "hello, test test", the part where i say "test test" is completely fine, whereas the "hello" has a low probability as usual.

This is on both the v4 and v5 model. Is this expected behavior?

**Comment by snakers4** [comment]
Can you send an audio example?

**Reply by qwbarch** [reply]
Whoops, sorry for not providing an example. Here is an audio clip:
https://drive.google.com/file/d/1MvX4tMAhDjtzq3mI32JJgt2r5-VzPiaP/view?usp=sharing

On the v4 model, this will still have a low probability (around 0.1f-0.3f if I remember right), but on the v5 model it seems to think there's no speech detected at all

**Reply by snakers4** [reply, marked as answer]
Loading different versions like this:

```
  model, utils = torch.hub.load(repo_or_dir='snakers4/silero-vad:v3.1',
                                model='silero_vad',
                                force_reload=True,
                                onnx=USE_ONNX)

  (get_speech_timestamps,
  save_audio,
  read_audio,
  VADIterator,
  collect_chunks) = utils
```

<details>
<summary>v3.1</summary>

![image](https://github.com/user-attachments/assets/fe8e592f-235e-4d52-9dfa-32c6b1348db5)
</details>


<details>
<summary>v4.0</summary>

![image](https://github.com/user-attachments/assets/c6883d64-ed04-4ac7-b97f-55d35913ad33)

</details>

<details>
<summary>v5.1</summary>

![image](https://github.com/user-attachments/assets/5e28d3c7-3871-46fe-a702-3053b3604cd4)
</details>

So, in your particular case, I would suggest these params:

```
speech_timestamps = get_speech_timestamps(wav, model, sampling_rate=SAMPLING_RATE, visualize_probs=True, threshold=0.15)
```
